python/cuml/common/base.pyx

```diff
@@ -364,6 +367,14 @@ class Base(TagsMixin,
             return {'preserves_dtype': [self.dtype]}
         return {}
 
+    def _set_nvtx_annotations(self):
+        for function in ['fit', 'transform', 'predict']:
```

Comment:
```suggestion
        for function in ['fit', 'transform', 'predict', 'fit_predict', 'fit_transform']:
```

Those are common too, though I wonder for estimators that have other functions (like ARIMA or Bayes), it might be a good idea to be more general or allow re-defining what is decorated by the inheriting class, what do you think?